tests/test_issue_9356_bash_shim_noclobber.py

```
@@ -88,6 +90,89 @@ def _run_driver(tmp: Path) -> subprocess.CompletedProcess[str]:
+        brew_environment["HOMEBREW_NO_AUTO_UPDATE"] = "1"
+        formula = subprocess.run(
+            ["brew", "list", "--formula", "bash"],
+            env=brew_environment,
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+            check=False,
+            timeout=30,
+        )
+        if formula.returncode != 0:
+            subprocess.run(
+                ["brew", "install", "bash"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)